tekton/utils/pipeline.go

```diff
@@ -57,7 +57,8 @@ type Pipeline struct {
 // the Pipeline.
 // +kubebuilder:object:generate=true
 type ParameterizedPipeline struct {
-	Pipeline
+	Pipeline `json:",inline"`
```

Comment:
what does this change?

Reply:
without this tag, it the struct does not _inherit_ the fields from the extending struct.

Reply:
Not entirely sure what that means, but if you are sure we need it, sounds good :slightly_smiling_face: